src/main/resources/log4j.properties

```diff
@@ -0,0 +1,7 @@
+# This sets the global logging level and specifies the appenders
```

Comment:
since this is meant to be a library, should we only load this file into the classpath when building the Runner executable jar? and leave it out of the library jar